.github/ISSUE_TEMPLATE/release.md

```diff
@@ -31,7 +28,7 @@ assignees: ''
 
 - [ ] Move this issue to Test Ready.
 - [ ] Notify `@testers` in [#testing].
-- [ ] Test build: [Deploy to Staging], choosing the above tag.
+- [ ] Test build: Add a `staged` tag to the PR to deploy.
 
```

Comment:
We also need to remember to
```suggestion
- [ ] Merge the PR
```

I'm not sure when the best time will be. If it's linked to the release issue, it will cause it to become closed.